src/components/card/card.spec.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import TestRenderer from 'react-test-renderer';
+import Card from './card.component';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+import 'jest-styled-components';
+import Icon from '../icon';
+import OptionsHelper from '../../utils/helpers/options-helper/options-helper';
+
+describe('Card', () => {
+  describe('when the content is added as children', () => {
+    it('then that content should be rendered inside the component', () => {
+      const content = (
+        <div><span>content</span></div>
+      );
+      const wrapper = renderCard({
+        children: content
+      });
+
+      expect(wrapper.containsMatchingElement(content)).toBe(true);
+    });
+
+    describe.each(OptionsHelper.alignFull)('and the "spacing" prop is set to %s', (spacing) => {
```

Comment:
I think this should be `OptionsHelper.sizesRestricted` (which is `['small', 'medium', 'large']`), instead of `OptionsHelper.alignFull`.

Reply:
@grabkowski: Thanks for fixing this in f6b3f21 :thumbsup: